ci/scripts/get-build-type.sh

```diff
@@ -57,7 +58,18 @@ if [[ "$build_reason" = "PullRequest" ]]; then
     else
         echo "PR doesn't contain OTBN changes"
     fi
+
+    # Check if the commit has only CDC related changes (run-cdc.tcl,
+    # cdc_waivers*.tcl).
+    echo "Checking for changes in this PR other than to CDC waivers"
+    git diff --quiet "$merge_base" -- ':(attr:!cdc)*.tcl' && only_cdc_change=1 || true
```

Comment:
Taking another look at this, I think the `':(attr:!cdc)*.tcl'` check might be too specific. This is going to check for changes in all files that do not have `cdc` specified _and_ end in `.tcl`. Instead, we want to check for changes in all files that do not have `cdc` specified (regardless of their file extension).